addons/pos_mercado_pago/models/pos_payment_method.py

```diff
@@ -0,0 +1,115 @@
+import logging
+
+from odoo import fields, models, _
+from odoo.exceptions import AccessError, UserError
+
+from .mercado_pago_pos_request import MercadoPagoPosRequest
+
+_logger = logging.getLogger(__name__)
+
+
+class PosPaymentMethod(models.Model):
+    _inherit = 'pos.payment.method'
+
+    mp_bearer_token = fields.Char(
+        string="Production user token",
+        help='Mercado Pago customer production user token: https://www.mercadopago.com.mx/developers/en/reference')
+    mp_webhook_secret_key = fields.Char(
```

Comment:
is it ok that pos user can read this? `groups=` to protect the access?

Reply:
I'll check this

Reply:
No the xml is just visual, if it’s a secret, it should be forbidden at the python level and only be accessed with `sudo()` when needed. You can bypass xml restriction if calling in rpc.

Reply:
just add groups="point_of_sale.group_pos_manager" in the fields declaration, like in the XML

Reply:
@davidmonnom 
Quand je mets 
```
    mp_bearer_token = fields.Char(
        string="Production user token",
        help='Mercado Pago customer production user token: https://www.mercadopago.com.mx/developers/en/reference',
        groups="point_of_sale.group_pos_manager")
    mp_webhook_secret_key = fields.Char(
        string="Production secret key",
        help='Mercado Pago production secret key from integration application: https://www.mercadopago.com.mx/developers/panel/app',
        groups="point_of_sale.group_pos_manager")
````
Et que je me connecte avec Marc Demo (simple user je pense), voilà ce que j'ai:
![Capture d’écran 2024-04-15 à 14 17 26](https://github.com/odoo/odoo/assets/7986118/5d25b8c8-3e37-4f6a-94e3-f3ce08635252)

Reply:
Avec groups="point_of_sale.group_pos_user" c'est ok

Reply:
Ok with groups="point_of_sale.group_pos_manager"
I use sudo() where necessary